.github/workflows/build.yaml

```diff
@@ -0,0 +1,60 @@
+
+name: Build
+
+on:
+  # Run this workflow for pushes on all branches
+  push:
+    branches:
+      - '**'
+  # Run this workflow when a tag or branch is created
+  create:
+  # Run this workflow for pull requests
+  pull_request:
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.6, 3.7, 3.8]
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies and setup
+      run: |
+        pip install -r test_requirements.txt
+    - name: Run tests
+      run: py.test
+
+
+
+  publish_to_pypi:
+    # Only run this job if the run_tests job has succeeded, and if
+    # this workflow was triggered by the creation of a new tag
+    needs: run_tests
+    if: github.event_name == 'create' && github.event.ref_type == 'tag' && github.event.repository.fork == false
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build package and publish to PyPI
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Looks good. I'm assuming you got the API token for the LCOGT PyPI account?

Reply:
That's right.